### PR TITLE
destroy method included

### PR DESCRIPTION
**Pull request by prakashinim**
please review and let me know if i have anything to change!

### CODE REVIEW

**Comment by ShimaGunz** [comment]
looks good to me

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -8,6 +9,14 @@ def show
     @chef = User.find(params[:id])
```
Comment (nbouffard) [inline app/controllers/users_controller.rb:9]: This line is repeated twice, once here and once in the destroy. Can you create a set_chef method to keep the code dry? thanks

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -6,6 +6,9 @@
   <% if @chef == current_user %>
     <%= link_to 'Edit profile' %>
   <% end %>
+  <% if @chef == current_user %>
+    <%= link_to 'Delete profile' %>
```
Comment (nbouffard) [inline app/views/users/show.html.erb:10]: This link_to is incomplete. It needs a path to destroy and is also missing data: {turbo_method...... 
The destroy action doesn't work without it

**Review by nbouffard — CHANGES_REQUESTED** [review]
Overall well done, just needs a couple things added and you will be all good!

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (prakashinim) [reply app/controllers/users_controller.rb:9]: yes sure. will do that. thanks for the correction!

File: app/views/users/show.html.erb (hunk 2 above)
Comment (prakashinim) [reply app/views/users/show.html.erb:10]: ok. will include that aswell!

**Pull request closed without merging**